app/assets/stylesheets/config/_generator.scss

```diff
@@ -453,7 +453,7 @@
     null
   ),
   ('object', 'object-fit', ('cover': cover), (), false),
-  ('aspect', 'aspect-ratio', ('16-9': 16 / 9), (), false),
+  ('aspect', 'aspect-ratio', ('16-9': calc(16 / 9)), (), false),
```

Comment:
This is covered in your CSS generator PR, which I guess we're aiming to merge first

Reply:
indeed, ive just merged it, so resolving this one.